### PR TITLE
Add `setupmeta` package.

### PR DESCRIPTION
**Pull request by matthewwardrop**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by matthewwardrop** [comment]
@zsimic As discussed elsewhere, for you to review :).

**Comment by zsimic** [comment]
I am confirming that I am willing to be listed here, and will gladly help should issues arise

**Comment by matthewwardrop** [comment]
This is ready for review: @conda-forge/help-python.

**Comment by matthewwardrop** [comment]
@conda-forge/help

**Comment by matthewwardrop** [comment]
@conda-forge-admin, please ping team

File: recipes/setupmeta/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (npavlovikj) [inline recipes/setupmeta/LICENSE:1]: Looks like the license file is already in the PyPI tarball, so this file can be removed from here.

**Comment by npavlovikj** [comment]
@matthewwardrop , do you want to add yourself as a maintainer as well? I left a minor comment to remove the LICENSE file since it is included in the tarball - other than that the recipe looks good to me, so we can merge if afterwards.

**Comment by matthewwardrop** [comment]
@npavlovikj Thanks for the review. I've removed the LICENSE file, and updated meta.yaml accordingly (should I be adding the comment that the LICENSE file can now be found in the source tarball?). I also added myself as a maintainer, just in case :).

File: recipes/setupmeta/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  imports:
+    - setupmeta
+
+about:
+  home: "https://github.com/zsimic/setupmeta"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE (inside source tarball)
```
Comment (npavlovikj) [inline recipes/setupmeta/meta.yaml:33]: Sorry if I wasn't clear earlier - please leave `license_file: LICENSE` here. The path to the license file, LICENSE, is relative to the source directory, which is the tarball, where the license is.

Comment (npavlovikj) [reply recipes/setupmeta/meta.yaml:33]: Also, please remove `(inside source tarball)` since that field should only contain the name of the license file.

**Comment by npavlovikj** [comment]
Thank you for contributing @matthewwardrop 🎉

**Comment by matthewwardrop** [comment]
And thanks for reviewing and helping, @npavlovikj !

**Pull request merged**